src/widgets/User/UserMain/UserTabsContent/Follow/CommonFollow.tsx

```
@@ -0,0 +1,89 @@
+import {
+  FollowTableHead,
+  FollowTableRow,
+} from '@/widgets/User/UserMain/UserTabsContent/Follow/follow-table'
+import { SortByType } from '@/widgets/UsersList/UsersList'
+import { Pagination, Table, TableBody, Typography } from '@funnyteam/ui-kit'
+
+import s from './CommonFollow.module.scss'
+
+type Props = {
+  emptyTab: string
+  followQuery: Function
```

Biome

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)